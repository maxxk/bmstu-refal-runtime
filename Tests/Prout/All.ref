$ENTRY Go
{
	= <Prout 'Start text' 12 'Bu' 34 (Aaaa Ccc 'as') (' checking _ _  -_- !') Ggggg 'a' A ((())()) "Symbol1" "Symbol2">
}
