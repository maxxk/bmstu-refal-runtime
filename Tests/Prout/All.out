Start text12 Bu34 (Aaaa Ccc as)( checking _ _  -_- !)Ggggg aA ((())())Symbol1 Symbol2

